default.nix

```diff
@@ -0,0 +1,58 @@
+{ environment ? "default"
+}:
+let
+    pkgs = import ./nix/nixpkgs-pinned.nix {};
+
+    python = pkgs.pythonChannable;
+
+    pythonEnv = python.withPackages (p: [
+      p.bandit
+      p.black
+      p.mypy
+      p.pydantic
+      p.pylint
+      p.pytest
+      # TODO: actually add to overlay or nixpkgs
+      # p.parametrized
```

Comment:
Quick review, but this doesn't support some of the newer Python versions anyway. Or at least, it is somewhat broken.

We should prefer `pytest.mark.parametrized`.

Reply:
Noted, how would I go about adding it? I tried to google that string but the closest I got was `pytest.mark.parametrize` on the [docs](https://docs.pytest.org/en/6.2.x/parametrize.html) but I can't find any similar packages anywhere.

Reply:
That's the one. It's almost a drop in replacement. Just need to change the test that use it

Reply:
Okay then, I'll need to take a while to figure out more about it after lunch and meetings before I get comfortable with it.

Reply:
Oh god it's param**e**terized that's why I couldn't find it :angery: